domenetjenester/inngangsvilkar/src/main/java/no/nav/foreldrepenger/inngangsvilkaar/medlemskap/InngangsvilkårMedlemskap.java

```diff
@@ -32,7 +58,63 @@ public class InngangsvilkårMedlemskap implements Inngangsvilkår {
 
         var resultat = InngangsvilkårRegler.medlemskap(grunnlag);
 
-        return RegelResultatOversetter.oversett(VilkårType.MEDLEMSKAPSVILKÅRET, resultat);
+        var vilkårData = RegelResultatOversetter.oversett(VilkårType.MEDLEMSKAPSVILKÅRET, resultat);
+
+        try {
+            var behandling = behandlingRepository.hentBehandling(ref.behandlingId());
+
+            var nyUtledingResultat = utledNyttMedlemskapAksjonspunkt(ref);
+            logDiff(behandling, nyUtledingResultat);
+        } catch (Exception e) {
+            LOG.info("Medlemskap V2 feilet", e);
+        }
+
+        return vilkårData;
+
+    }
+
+    private static void logDiff(Behandling behandling, Set<MedlemskapAksjonspunktÅrsak> nyUtledingResultat) {
+        var utledetAksjonspunkt = behandling.getAksjonspunkter()
```

Comment:
Vi bør logge alle utfall for å kvantifisere + ytelse i alle tilfelle
* Like, ingen avklaring
* Like, samme avklaringer
* Ulike

Reply:
Lagt inn logging for de tilfellene du nevner med ytelse